### DISCUSSION TITLE
How to connect to redis or pg ?

### DISCUSSION THREAD

**Discussion by HosMercury** [maxcountryman/axum-sessions#40] [Q&A]
where is the docs for connecting to redis or pg ?

**Comment by maxcountryman** [comment]
Have you seen the [async-sqlx-session](https://github.com/maxcountryman/axum-sessions/tree/main/examples/async-sqlx-session) example? You can use Postgres `async-sqlx-session`.

**Reply by dignifiedquire** [reply]
that unfortunately doesn't work with tokio, as it depends on async-std

**Reply by maxcountryman** [reply]
@dignifiedquire I'm not sure I'm following. For example: https://github.com/maxcountryman/axum-sessions/blob/86537bcc0803d49ede22e64da46c21b404891aff/examples/async-sqlx-session/Cargo.toml#L23. Please do check the examples.

**Reply by dignifiedquire** [reply]
Thanks, I couldn't get the feature overrides to work last time I tried, and `async-sqlx-session` doesn't expose any features to enable tokio directly: https://github.com/jbr/async-sqlx-session/blob/main/Cargo.toml#L17 
Will try again following the example.

**Reply by dignifiedquire** [reply]
found the issue, the currently release of `async-sqlx-session` pulls in sqlx@0.5, making the override not work if you are using sqlx@0.6

**Reply by maxcountryman** [reply]
@dignifiedquire glad you were able to isolate the issue. It'd be great if we could address with a PR.

**Reply by Arqu** [reply]
I got an open PR at https://github.com/jbr/async-sqlx-session/pull/33 as you're aware @maxcountryman. 
Bumping `async-session` is not a hard requirement for `async-sqlx-session` itself, but many of the deps are stale on `async-session@3.0.0` and make for hard conflicts in combination with other libraries a project might pull in.

**Reply by maxcountryman** [reply]
@Arqu we've just released tower-sessions if you'd like to try that.

**Comment by HosMercury** [comment]
but this is sqlite not pg?

**Reply by HosMercury** [reply]
There?

**Comment by maxcountryman** [comment]
We've release [tower-sessions](https://github.com/maxcountryman/tower-sessions) 0.1.0 which includes support for Redis.
